### PR TITLE
Enhanced logging during DEV mode

### PR DESCRIPTION
**Pull request by amrfaissal**
This enhances the logging during DEV mode. See below:

beego_color

It supports Windows too:

beego_color_win

### CODE REVIEW

**Comment by astaxie** [comment]
does this support windows?

**Comment by amrfaissal** [comment]
Now it does.

File: logs/logger.go [hunk 1]
```diff
@@ -18,6 +18,9 @@ import (
 	"io"
 	"sync"
 	"time"
+	"github.com/shiena/ansicolor"
```
Comment (astaxie) [inline logs/logger.go]: I don't like import third package in beego. I want to keep beego independence.

**Comment by astaxie** [comment]
@amrfaissal build failed. NewAnsiColorWriter not existed

**Comment by amrfaissal** [comment]
@astaxie Yes, I'm working on it.

**Comment by amrfaissal** [comment]
@astaxie FYI, the bug is fixed.

**Comment by astaxie** [comment]
awesome

**Pull request merged**